src/d2l-rubric-levels-mobile.js

```diff
@@ -928,9 +929,13 @@ const component = {
 	},
 
 	_handleMove: function(event, delta) {
-		event.preventDefault();
-		event.stopPropagation(delta);
-		this._moveSelection(delta);
+		// Delay using afterNextRender so the _blurHandler in
+		// d2l-rubric-feedback will fire before the below statements.
+		afterNextRender(this, () => {
+			event.preventDefault();
+			event.stopPropagation(delta);
```

Comment:
```suggestion
			event.stopPropagation();
```
this was a typo on my end, stopPropagation doesn't take args

Reply:
Ahh ok. Well thats good to know.
https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation

Reply:
Is it necessary to re-test this? I'm in the middle of setting up Bsi for 4.x so I can't really test it. If we don't need to test then I am fine to just commit it. As you said event.stopPropagation doesn't take any parameters so removing this variable shouldn't change any behaviour. 
Since javascript is loosely typed then I don't think we should really need to re test this.
https://stackoverflow.com/questions/11107823/what-happens-if-i-dont-pass-a-parameter-in-a-javascript-function#:~:text=JavaScript%20functions%20can%20be%20invoked,any%20type%20to%20any%20function.

Reply:
Shouldn't be necessary to re-test. It's interesting that stopPropagation doesn't panic at all with the extra arg since some DOM methods are really picky about the props that you pass in. It won't change anything about the behavior though since it's not supposed to be receiving anything in the first place.

On top of that I'd doubt stopPropagation is even necessary here since the radio buttons shouldn't have any children that receive key events. I must've just added it as a failsafe on top of preventDefault. That's something I'd have to re-test though since native html inputs can be weird at times

Reply:
Sure that works for me. I'll commit this.